e2e/playwright/viewer/exclude.tests.json

```diff
@@ -1 +1,3 @@
-{}
+{
+  "C284636": "https://alfresco.atlassian.net/browse/ACS-5607"
```

Comment:
you are adding the test and disabling it here for the playwright

Reply:
yes, this test I added, but failing due to timeout , taking longer to search result appear , so comment till we start using ES results.